.eslintrc.json

```diff
@@ -28,11 +28,13 @@
     "comma-dangle": ["error", "always-multiline"],
     "prefer-template": ["error"],
     "react/button-has-type": ["error"],
+    "react/jsx-max-depth": ["off"],
```

Comment:
I'm glad you added it as "off" to be more explicit. I could see us enabling it at some point but this is good for right now.